Vecherskiys/02/pic/Python.add/Task2.3.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <HTML>
  <HEAD>
+ <meta name="Description" content="About Python">
```

Comment:
meta tag for charset https://www.w3schools.com/tags/att_meta_charset.asp

Reply:
Ok